test/cpp_extensions/open_registration_extension/pytorch_openreg/__init__.py

```diff
@@ -1,9 +1,11 @@
 import types
 
-import torch
-
 # Create our python implementation dict so that the C++ module
 # can access it during its initialization and also register aten impls.
+from typing import Union
+
+import torch
```

Comment:
Not sure why you're moving this between the comment and the import corresponding to it? But you most likely want to revert that change.